app/controllers/time_sheets_controller.rb

```diff
@@ -25,8 +40,8 @@ def update
   def set_time_sheet
     @time_sheet = TimeSheet.find(params[:id])
   end
-
+  
```

Comment:
Kannst du das Leerzeichen noch entfernen? :) 
Sehr cool in Zeile 45 mit dem map :+1: